golden_test/README.md

```diff
@@ -0,0 +1,22 @@
+# Golden tests
+Golden tests, also called screenshot tests, let's you write tests to easily find UI regressions. It
+compares the UI to saved goldens screenshots, and if there is a pixel difference the tests will
+fail. It gives us confidence that code changes does not cause UI regressions.
+
+## Updating goldens
+An expected change to the UI will also cause the golden tests to fail. You will then need to update
```

Comment:
```suggestion
An expected change to the UI will also cause the golden tests to fail. If this is unintended, you will need to change your code to prevent the UI change. If it was intended, you will then need to update
```

Reply:
Done.